### PR TITLE
pulling the timezone off the HTML element if present

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: test/d2l-localize-behavior.html [hunk 1]
```diff
@@ -200,6 +208,18 @@
...
 
 					it('should format a time using specified format', function() {
+						var val = elem.formatTime(date, {format: 'full'});
+						expect(val).to.equal('5:13 PM Canada - Toronto');
+					});
+
+					it('should not fail if timezone data is missing', function() {
+						htmlElem.removeAttribute('data-timezone');
+						var val = elem.formatTime(date, {format: 'full'});
+						expect(val).to.equal('5:13 PM ');
```
Comment (ryantmer) [inline test/d2l-localize-behavior.html:218]: Not really related, but that extra space seems a bit weird, is that a bug in `formatTime`? ("bug" might be a strong word for an extra space, granted 😜)

**Review by ryantmer — APPROVED** [review]
Neat! 🙌 What's setting this attribute again? Pretty sure I was on the review for it, but that was pre-break 😛

File: test/d2l-localize-behavior.html (hunk 1 above)
Comment (dlockhart) [reply test/d2l-localize-behavior.html:218]: Yeah it's kind of a bug in that it has no timezone name so it just appends a space and then nothing. It's "existing" behaviour, but agreed that its wrong.

**Comment by dlockhart** [comment]
There's a plugin in the monolith that will set this attribute on the HTML element, and ifrau will marshall it between the host page and an iFRA client.

**Pull request merged**